### ISSUE TITLE
Translate into French (Français)

### ISSUE THREAD

**Issue by seb86** [co-cart/co-cart#277]
Would like to have CoCart translated in French.

The original language is written in English specifically (en_US).

Simply [download the original POT](https://raw.githubusercontent.com/co-cart/co-cart/master/languages/cart-rest-api-for-woocommerce.pot) file and create a new translation.

> Right click and press "Save link as". Then add at the end of the filename ".pot" and select "All files" in the dropdown and the save.

## How to translate.

Easiest way to do it as this issue is for Hacktoberfest is to download the recommended program (FREE). Then create new translation from POT template.

![Create new from POT template](https://user-images.githubusercontent.com/497406/137648429-1f93c450-345e-4b1e-abf1-4a86eeb521b3.png)

Then select or type "French (France)" and press OK.

![Select language for PoEdit](https://user-images.githubusercontent.com/497406/137648451-710bc30a-89a7-4a8d-a692-e5845ca24bb8.png)

After that, start translating. When your done translating, go to "File -> Save as". You will see it only sets the filename as "fr_FR". It needs to be "cart-rest-api-for-woocommerce-fr_FR"

Then upload both "cart-rest-api-for-woocommerce-fr_FR" files saved here.

I will then audit the translation and provide any feedback before marking the issue complete.

Recommended program: https://poedit.net/

Save file as "cart-rest-api-for-woocommerce-fr_FR"

**Comment by evokelektrique** [comment]
Would you like to have other languages translations like Persian ?

**Comment by seb86** [comment]
@evokelektrique Yes, please do translate into what ever language you know. Same instructions, just make sure you select the correct language before you start.

**Comment by evokelektrique** [comment]
@seb86 Alright then, I'll keep you posted.
